src/components/NeighborhoodSelector.js

```diff
@@ -36,12 +36,13 @@ const NeighborhoodSelector = ({ neighborhood, setNeighborhood }) => {
 
   return (
     <div>
-      <Header>
-        <img alt="title" src={'an image'} />
+      <Header title={"Search"}>
+        {/* What does line 41-45 do? */}
+        {/* <img alt="title" src={'an image'} />
```

Comment:
I've been confused about this too.  I don't think we need Event  Title on this page at all, right?  And actually, for the neighborhood selector page maybe weldon't need Share .  Or it should just... share the home page of the app maybe?  But I have been wondering about these lines.  Maybe we should just remove them.  I'm in favor of removing code instead of commenting it, if it's of no use.  Git history can be used to restore it